subjects/weareunique/README.md

```diff
@@ -0,0 +1,57 @@
+## weareunique
+
+### Instructions
+
+Write a function that takes two strings and returns the number of characters that are not included in both, without doubles.
```

Comment:
- The word strings should be formatted --> `strings`. 
-  Add missing space in between line 5 and line 6.

Reply:
fixed